deployments/charts/kuma/values.yaml

```diff
@@ -664,6 +664,13 @@ hooks:
   containerSecurityContext:
     readOnlyRootFilesystem: true
 
+  # -- ebpf-cleanup hook needs write access to the root filesystem to clean ebpf programs
+  ebpfCleanup:
+    # -- Security context at the pod level for crd/webhook/cleanup-ebpf
+    podSecurityContext: {}
```

Comment:
Can we set `runAsNonRoot: true` here?

Reply:
We can't as it's removing ebpf frograms from `/sys/fs/bpf`

Reply:
Then same as above with disallowing it as `true` IMO